README.md

```diff
@@ -136,6 +136,11 @@ Change your `.env` file to have:
 PROTOCOL=https
 ```
 
+### Setting up as OpenID Connect provider
+- Run `yarn generate-rsa-key-base64`
+- Copy generated base64 string to .env file: `OIDC_JWK_KEYS=base64key`
+- You can add multipke keys by separating them with a comma
```

Comment:
Typo on "multipke" and curious why one would generate multiple keys? Do they get rotated over?